app/controllers/api.js

```diff
@@ -154,8 +156,10 @@ API.createBike = function (req, res){
         var bike = new Bike();
```

Comment:
You can go ahead and remove this line and it's closure: 
Bike.findOne().sort('-bikeID').select('bikeID').exec(function (err, object) {

That was used to get the next bikeID, but were just giving it stuff now.